torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -478,15 +480,43 @@ void IRPrinter::handle(const ReductionOp* rop) {
 }
 
 void IRPrinter::handle(const BroadcastOp* bop) {
```

Comment:
Shouldn't we follow more closely to ReductionOp to first check if it's been lowered to figure out how we're going to print it?
```
  bool lowered = rop->out()->getValType() == ValType::TensorIndex;
  if (!lowered) {
    os << rop->out() << " = reduction( " << rop->in()
       << ", op = " << rop->getReductionOpType()
       << ", initial value = " << rop->init() << " )\n";
    return;
  }
```

It looks to me like you're going to print `broadcast::blockBroadcast<...` even if it hasn't been lowered.

Reply:
Thanks. That's right. Fixed.